ios/FluentUI/TextField/FluentTextField.swift

```diff
@@ -0,0 +1,303 @@
+    lazy public var tokenSet: TextFieldTokenSet = .init(state: { [weak self] in
+        return self?.state ?? .unfocused
+    })
+
+    /// UIImage used in the leading UIImageView. If this is nil, the leading UIImageView will be hidden.
+    @objc public var leadingImage: UIImage? {
+        didSet {
+            if let leadingImage {
+                leadingImageView.image = leadingImage
+                leadingImageContainerView.isHidden = false
+            } else {
+                leadingImageContainerView.isHidden = true
+            }
+        }
+    }
+
+    /// String used in the title label. If this is nil, the title label will be hidden.
+    @objc public var titleText: String? {
+        didSet {
+            if let titleText {
+                titleLabel.text = titleText
+                titleLabel.isHidden = false
+            } else {
+                titleLabel.isHidden = true
+            }
+        }
+    }
+
+    /// String representing the input text.
+    @objc public var inputText: String? {
+        get {
+            return textfield.text
+        }
+        set {
+            textfield.text = newValue
+        }
+    }
+
+    /// String representing the placeholder text.
+    @objc public var placeholder: String? {
+        didSet {
+            textfield.attributedPlaceholder = attributedPlaceholder
+        }
+    }
+
+    /// String used in the assistive text label. If this is nil, the assistive text label will be hidden. If the `error`
+    /// property of the `FluentTextField` is set, the `localizedDescription` from `error`
+    /// will be displayed instead.
+    @objc public var assistiveText: String? {
+        didSet {
+            updateAssistiveText()
+        }
+    }
+
+    /// The closure for the action to be called in response to the textfield's `.editingChanged` event.
+    @objc public var onEditingChanged: ((FluentTextField) -> Void)?
+
+    /// The closure for the action to be called in `textFieldDidBeginEditing`.
+    @objc public var onDidBeginEditing: ((FluentTextField) -> Void)?
+
+    /// The closure for the action to be called in `textFieldDidEndEditing`.
+    @objc public var onDidEndEditing: ((FluentTextField) -> Void)?
+
+    /// The closure for the action to be called in `textFieldShouldReturn`. The return value of `onReturn`
+    /// will be returned in `textFieldShouldReturn`.
+    @objc public var onReturn: ((FluentTextField) -> Bool)?
+
+    /// The `FluentTextInputError` containing the `localizedDescription` that will be
+    /// displayed in the assistive text label.
+    @objc public var error: FluentTextInputError? {
+        didSet {
+            updateState()
+            updateAssistiveText()
+        }
+    }
+
+    // Hierarchy:
+    //
+    // imageTextStack
+    // |--leadingImageContainerView
+    // |--|--leadingImageView
+    // |--textStack
+    // |--|--titleLabel
+    // |--|--textfield
+    // |--|--separator
+    // |--|--assistiveTextLabel
+    @objc public init() {
```

Comment:
is this now designated? UIView initialzation alwasy have initwithframe as defaut